pkg/gce-pd-csi-driver/controller_test.go

```diff
@@ -3304,13 +3313,61 @@ func TestCleanSelfLink(t *testing.T) {
 			in:   "https://www.partnerapis.com/compute/alpha/projects/project/zones/zone/disks/disk",
 			want: "projects/project/zones/zone/disks/disk",
 		},
+		{
+			name: "alpha project",
+			in:   "https://www.googleapis.com/compute/alpha/projects/alphaproject/zones/zone/disks/disk",
```

Comment:
Nit: another nice test would be something like project name = betabeta

Reply:
done